configuration/scripts/cice.launch.csh

```diff
@@ -130,7 +130,7 @@ EOFR
 endif
 
 #=======
-else if (${ICE_MACHINE} =~ daley* || ${ICE_MACHINE} =~ banting*) then
```

Comment:
same comment here. in fact I'm realizing that this is wrong for the serial case, as if we don't use `aprun` the model will run on the MOM node instead of the computes...